Log/lambda_function.py

```diff
@@ -275,6 +280,14 @@ def send_entry(s, log_entry):
 
     # Send to Datadog
     str_entry = json.dumps(log_entry)
+    
+    #Scrub ip addresses if activated
+    if is_ipscrubbing:
+        try:
+            str_entry = re.sub('\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}',"xxx.xxx.xxx.xx",str_entry)
```

Comment:
Could you `re.compile` this regex like we do with cloudtrail?

Reply:
@tmichelet done